src/vs/workbench/services/workingCopy/electron-sandbox/workingCopyBackupTracker.ts

```diff
@@ -303,7 +303,7 @@ export class NativeWorkingCopyBackupTracker extends WorkingCopyBackupTracker imp
 	}
 
 	private doSaveAllBeforeShutdown(modifiedWorkingCopies: IWorkingCopy[], reason: SaveReason): Promise<void>;
-	private doSaveAllBeforeShutdown(includeUntitled: boolean, reason: SaveReason): Promise<void>;
+	private doSaveAllBeforeShutdown(includeAllUntitled: boolean, reason: SaveReason): Promise<void>;
```

Comment:
Interestingly I only see 2 usages of `doSaveAllBeforeShutdown`: the one with an array of modified working copies and the other with `false`. So I think over time the file has changed to no longer need the `includeUntitled: true` choice. I would then go ahead and clean this method up accordingly.

To address your case, I guess it would require changes elsewhere to include scratchpads for saving? Or maybe that is already the case?